# Exercício 5 - Passando promessa de pai para filho

O problema está na execução do segundo `then`, onde o resultado da variavél `data` será `undefined`, porquê o valor da variável `data`, disponível na primeira chamada, não é passado adiante. Para corrigir isso, basta fazer o seguinte ajuste:

```javascript
//...
promise
.then((data) => {
  console.log(`resultado positivo: ${data}`)
  return data
})
.then((data) => console.log(`resultado positivo 2: ${data}`))
.catch((data) => console.log(`resultado negativo: ${data}`));
```